contributor-docs/release-guide.md

```diff
@@ -528,9 +528,34 @@ The following should be confirmed:
 - [ ] There is a commit not on the release branch with the version adjusted.
 - [ ] The RC tag points to that commit.
 
+### Create a draft, pre-release Github release for the RC Tag
+
+TODO: Automate these steps as a github action.
+
+If this is for the first release candidate, create a new, draft, pre-release Github release.
+
+Go to https://github.com/apache/beam/releases/new to start creating a Github release.
+
+If this is for subsequent release candidates, re-use the existing Github release for this version.
+Do not create a new release.
+
+Update the Release tag to the current RC Tag.
```

Comment:
Maybe bullets? (this will render as a paragraph, right? so the steps may not be called out as loudly as you want)

Reply:
Agreed. Done.